quail/processes/wps_climdex_id.py

```diff
@@ -0,0 +1,123 @@
+from rpy2 import robjects
+from pywps import Process, LiteralInput
+from pywps.app.Common import Metadata
+from pywps.inout.formats import Format
+
+from wps_tools.utils import log_handler, collect_args, common_status_percentages
+from wps_tools.io import log_level
+from quail.utils import get_package, logger, load_rdata_to_python, save_python_to_rdata
+from quail.io import climdex_input, ci_name, output_path, rda_output
+
+
+class ClimdexID(Process):
```

Comment:
There is so much overlap between this an summer days. Would it make more sense for us to make a "days" action that takes in the type of day as a flag? In this case either summer or icing? I think doing it that way would save us a lot of code.

Reply:
@nikola-rados good idea, how would I do a flag?

Reply:
Also do you mean one process?

Reply:
Just have a `boolean` input for each type, or a single input with `string` choices. I think I would prefer the latter.

Reply:
Yes I did mean that.

Reply:
Okay well I might as well do #9 tropical nights as well in this PR?

Reply:
No, lets keep each PR to one issue. We want to try and merge bite-sized chunks of code. If we start trying to do too much at once it will leads to issues.